core-field-editors/README.md

```diff
@@ -0,0 +1,32 @@
+Contentful Core Field Editors
+=============================
+
+This folder contains the source code to some of the Contentful Web App’s builtin
+field editors. The code targets the [Contentful UI Extension
+API][ui-extension-sdk]. This allows you to modify the source code, upload the
+editors as extensions to Contentful and use them in our App.
+
```

Comment:
Proposal: 
"This folder contains the source code for field editors builtin within the Contentful Web Application.
Since these are developed using the UI Extensions SDK, this will allow you to understand how each component works, and create your own extensions based on existing Contentful components' source rather than starting from scratch."